tests/framework/jailer.py

```diff
@@ -156,35 +156,41 @@ def chroot_ramfs_path(self):
         """Return the MicroVM chroot ramfs subfolder path."""
         return os.path.join(self.chroot_path(), self.ramfs_subdir_name)
 
+    @property
+    def uses_ramfs(self):
+        """Is this jailer using ramfs?"""
+        return self._ramfs_path is not None
+
     def jailed_path(self, file_path, create=False, create_jail=False):
         """Create a hard link or block special device owned by uid:gid.
 
         Create a hard link or block special device from the specified file,
         changes the owner to uid:gid, and returns a path to the file which is
         valid within the jail.
         """
-        file_name = os.path.basename(file_path)
-        global_p = os.path.join(self.chroot_path(), file_name)
+        file_path = Path(file_path)
+        chroot_path = Path(self.chroot_path())
+        global_p = chroot_path / file_path.name
```

Comment:
I think using `/` here is unclear, it would be useful to add a comment or use an inbuilt function instead.